stablehlo/tests/interpret_reverse.mlir

```diff
@@ -0,0 +1,53 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
```

Comment:
Do we need all the three tests here or just one is sufficient, similarly to `ops_stablehlo.mlir` which has just one positive test?